static/js/main.js

```diff
@@ -241,17 +243,23 @@ function addScreenshot(screenshot) {
                             value='${time_input_value}' />
                 </div>
                 <div>
-                    <button onclick='flipImage("${screenshot.id}")'>Flip Image</button>
+                    <span>
+                        <button onclick='flipScreenshot("${screenshot.id}", axis="x")'>Vertical Flip</button>
+                        <button onclick='flipScreenshot("${screenshot.id}", axis="y")'>Horizontal Flip</button>
+                    </span>
                 </div>
+                <div>
+                    <button onclick='delete_screenshot("${screenshot.id}")'>Delete</button>
```

Comment:
This PR also has deletion, but was not indicated anywhere.

Reply:
Yeah, sorry, mentioned it in Slack, didn't realize I needed to do a different branch for a different PR until after committing.